verticapy/tests/utilities/test_utilities.py

```diff
@@ -12,49 +12,276 @@
+        base.cursor.execute("DROP TABLE IF EXISTS public.verticapy_table_test")
+        base.cursor.execute("CREATE TABLE verticapy_table_test AS SELECT 1;")
+        drop("verticapy_table_test", base.cursor)
+        base.cursor.execute(
+            "SELECT table_name FROM columns WHERE table_name = 'verticapy_table_test' GROUP BY 1;"
+        )
+        result = base.cursor.fetchall()
+        assert result == []
+        # view
+        base.cursor.execute("DROP VIEW IF EXISTS public.verticapy_view_test")
+        base.cursor.execute("CREATE VIEW verticapy_view_test AS SELECT 1;")
+        drop("verticapy_view_test", base.cursor)
+        base.cursor.execute(
+            "SELECT table_name FROM view_columns WHERE table_name = 'verticapy_view_test' GROUP BY 1;"
+        )
+        result = base.cursor.fetchall()
+        assert result == []
+        # text index
+        base.cursor.execute(
+            "CREATE TEXT INDEX drop_index ON drop_data (id, transportation);"
+        )
+        drop("drop_index", base.cursor)
+        try:
+            base.cursor.execute("SELECT * FROM drop_index;")
+            fail = False
+        except:
+            fail = True
+        assert fail
+        # model
+        base.cursor.execute("DROP MODEL IF EXISTS public.verticapy_model_test")
+        base.cursor.execute(
+            "SELECT NAIVE_BAYES('public.verticapy_model_test', 'public.drop_data', 'transportation', 'gender, cost');"
+        )
+        drop("verticapy_model_test", base.cursor)
+        base.cursor.execute(
+            "SELECT model_name FROM models WHERE model_name = 'verticapy_model_test' GROUP BY 1;"
+        )
+        result = base.cursor.fetchall()
+        assert result == []
+        # verticapy model
+        with warnings.catch_warnings(record=True) as w:
+            drop("verticapy", base.cursor, method="schema")
+        create_verticapy_schema(base.cursor)
+        model = KNeighborsClassifier("verticapy_model_test", base.cursor)
+        model.fit("public.drop_data", ["gender", "cost",], "transportation")
+        drop("verticapy_model_test", base.cursor)
+        base.cursor.execute(
+            "SELECT model_name FROM verticapy.models WHERE model_name = 'verticapy_model_test' GROUP BY 1;"
+        )
+        result = base.cursor.fetchall()
+        assert result == []
+        drop("verticapy", base.cursor, method="schema")
+        # geo index
         world_copy = world_vd.copy()
         world_copy["id"] = "ROW_NUMBER() OVER (ORDER BY pop_est)"
         result = create_index(world_copy, "id", "geometry", "world_polygons", True)
-        result = drop_geo_index(
-            "world_polygons", world_vd._VERTICAPY_VARIABLES_["cursor"],
+        drop(
+            "world_polygons", base.cursor,
+        )
+        try:
+            describe_index(
+                "world_polygons", base.cursor, True,
+            )
+            fail = False
+        except:
+            fail = True
+        assert fail
```

Comment:
It is better the pytest.raises for asserting exceptions.

Reply:
corrected